README.md

```diff
@@ -55,6 +55,26 @@ end
 
 See also: [test/reline/yamatanooroti/multiline_repl](https://github.com/ruby/reline/blob/master/test/reline/yamatanooroti/multiline_repl)
 
+## Contributing
+
+Bug reports and pull requests are welcome on GitHub at https://github.com/ruby/reline.
+
+### Run tests
+
+```bash
+WITH_VTERM=1 bundle install
+
+bundle exec rake test
+WITH_VTERM=1 bundle exec rake test_yamatanooroti
```

Comment:
A weird thing is that I always get:

```
The git source https://github.com/ruby/yamatanooroti.git is not yet checked out. Please run `bundle install` before trying to start your application
```

I'm sure the condition is entered and `bundle install` succeeded:

```
be rake test
Using vterm-gem
The git source https://github.com/ruby/yamatanooroti.git is not yet checked out. Please run `bundle install` before trying to start your application
```

I had no problem running it after removing the `if` condition from Gemfile though. Any idea?

Reply:
> A weird thing is that I always get:

I was wondering about that message as well.
It seems to me that the warning is because the contents of `Gemfile.lock` changes depending on whether `WITH_VTERM=1` is set or not.

> I had no problem running it after removing the if condition from Gemfile though. Any idea?

I think this `if` was included in ruby/ruby to prevent yamatanooroti tests from running. 
Sorry if I'm wrong, I have a hazy memory.

Reply:
I think it could just be my terminal was in a bad state. It seems to work fine now 🤔 

> I think this if was included in ruby/ruby to prevent yamatanooroti tests from running.

Probably not because Gemfile is not copied to `ruby/ruby`.

I guess it's because the way its CI is setup: We only install libvterm on ubuntu on CI but we run non-yamatanooroti tests on all platforms (including macos and windows). So if we remove that condition, bundle install would fail on those platforms.

Since we're requiring contributors to install libvterm now, we "may" remove that condition. BUT, it means we will need to install libvterm on every CI environment, which could be a big headache. So I don't think we need to do this yet.